### ISSUE TITLE
🐛 Incompatible with underscore

### ISSUE THREAD

**Issue by fregante** [rome/tools#1475]
`rome check` fails if `underscore` exists.

## Environment

```
Environment Variables
  Object {
    LANG: "en_US.UTF-8"
    TERM: "xterm-256color"
    TERM_PROGRAM: "Apple_Terminal"
  }

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  15.14.0

Platform
  darwin x64 20.4.0

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 1
    columns: 100
    cursor: false
    hyperlinks: false
    isTTY: false
    unicode: false
  }

Client Flags
  Object {
    clientName: "cli"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 28_508_160
      pid: 18_083
      uptime: 0.206555484
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 28_508_160
        ownedBytes: 0
        ownedFileCount: 0
        pid: 18_083
        uptime: 0.206498949
      }
    ]
  }

```

## Steps To Reproduce

```sh
npm init --yes
npm install underscore
npx rome init
npx rome check
```

```

 node_modules/underscore/package.json:26:19 parse/manifest ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected a string at exports["."].["import"].browser

    24 │       "import": {
    25 │         "module": "./modules/index-all.js",
  > 26 │         "browser": {
       │                    ^
  > 27 │           "production": "./underscore-esm-min.js",
  > 28 │           "default": "./underscore-esm.js"
  > 29 │         },
       │ ^^^^^^^^^
    30 │         "node": "./underscore-node.mjs",
    31 │         "default": "./underscore-esm.js"

  ⚠ Our internal value has been modified since we read the original source

 node_modules/underscore/package.json:34:19 parse/manifest ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected a string at exports["."].require.browser

    32 │       },
    33 │       "require": {
  > 34 │         "browser": {
       │                    ^
  > 35 │           "production": "./underscore-umd-min.js",
  > 36 │           "default": "./underscore-umd.js"
  > 37 │         },
       │ ^^^^^^^^^
    38 │         "node": "./underscore-node.cjs",
    39 │         "default": "./underscore-umd.js"

  ⚠ Our internal value has been modified since we read the original source

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

✖ Found 2 problems
```

**Comment by fregante** [comment]
Generally speaking, it probably should not completely fail if a (sub-)dependency doesn’t match Rome’s expectations. This was the first time I tried Rome.

**Comment by dctalbot** [comment]
Did a little bit of research, seems rome might not yet support [conditional exports](https://nodejs.org/api/packages.html#packages_conditional_exports) and the `normalizeExportsField` function is stubbed at the moment

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
